src/monodroid/monodroid.props

```diff
@@ -14,8 +14,8 @@
     <_CmakeMxeCommonFlags>$(_CmakeCommonFlags) $(_CommonDefines) -DENABLE_NDK=OFF</_CmakeMxeCommonFlags>
 
     <_CmakeMxeCommonFlags Condition=" '$(_MingwDependenciesRootDirectory)' != '' ">$(_CmakeMxeCommonFlags) -DMINGW_DEPENDENCIES_ROOT_DIR=$(_MingwDependenciesRootDirectory)</_CmakeMxeCommonFlags>
-    <_CmakeMxeCommonFlags32>$(_CmakeMxeCommonFlags) -DCMAKE_TOOLCHAIN_FILE=$(_CmakeMxeToolchainBasePath)-32.cmake -DMINGW_TARGET_32=1 -DMINGW_ZLIB_LIBRARY_PATH=&quot;$(MingwZlibRootDirectory32)\lib\$(MingwZlibLibraryName)&quot;</_CmakeMxeCommonFlags32>
```

Comment:
What's the rationale/commit message content for this & related changes? It doesn't appear to be directly related to Catalina, is it?

Reply:
The `-DMINGW_ZLIB_LIBRARY_PATH` parameter was causing issues now that we've removed the xamarin-windeps homebrew recipe.  I've updated the PR description with the two build errors I hit while working with this new pool initially.